installer/x86_64/install.sh

```diff
@@ -55,7 +53,17 @@ ONIE_PLATFORM_EXTRA_CMDLINE_LINUX=""
 # Default var/log device size in MB
 VAR_LOG_SIZE=4096
 
-[ -r platforms/$onie_platform ] && source platforms/$onie_platform
+
+# simple detect whether script was run in SONiC
```

Comment:
Before I added support for installation form SONiC there were two options ONIE and some Linux. I'm not aware about the installation from Linux (some other platforms?) so I tried to keep the old code under "if != sonic" and add new "under if = sonic".